src/tests/stats/test_median.fypp

```diff
@@ -117,12 +117,6 @@ contains
         call check(error, median(d2odd_${k1}$),  1._dp&
                     , 'median(d2odd_${k1}$): uncorrect answer'&
                     , thr = dptol)
-        if (allocated(error)) return
```

Comment:
Here, did you remove one-too-many `if(allocated(error))` ?

Reply:
It is the last test, so it will return anyway. I can add it back, if it clarifies things.

Reply:
Oh makes sense -- I don't mind, as you like.